regression/rt_gk_lbo_relax_varnu_1x2v_p1.c

```diff
@@ -378,13 +377,14 @@ main(int argc, char **argv)
     .collisions =  {
       .collision_id = GKYL_LBO_COLLISIONS,
       .normNu = true,
-      .bmag_mid = ctx.B0,
-      .self_nu_fac = ctx.self_nu_fac,
-      .cross_nu_fac = ctx.cross_nu_fac_tophat,
+      .n_ref = ctx.n0,
+      .T_ref = ctx.vt*ctx.vt*ctx.mass,
+      .hbar = 1.0,
+      .eps0 = 1.0,
+      .eV = 1.0,
```

Comment:
@akashukla Since reg tests serve as an example to others, please comment on each of these lines describing their meaning, and why or why not they need to be specified.